### ISSUE TITLE
Pywatchman docs

### ISSUE THREAD

**Issue by tim-mccurrach** [facebook/watchman#988]
### Are there any docs for pywatchman specifically?

The PyPi page for pywatchman directs you to this repo, which has a link to some generic watchman docs. But I can't find anything about pywatchman itself. I appreciate the source-code is available for one to inspect, but figuring it out whilst you're still getting to grips with watchman itself can be tricky.

Even just a few examples demonstrating some basics would be incredibly helpful. Something similar to the page provided for the NodeJS docs. I would be very happy to contribute if such a notion was acceptable in principle - although I would put myself in the category I described above, so there would definitely be others better qualified to do so.

(I apologise if I am missing some obvious part of the docs. I have tried to look carefully through them).

**Comment by chadaustin** [comment]
No, there is no Python-specific documentation at this time. I'd be happy to review and accept a PR if you wanted to figure out the example you wish you had!

**Comment by tim-mccurrach** [comment]
I'll see if I can put something together soon :)
